addOns/pscanrules/src/main/resources/org/zaproxy/zap/extension/pscanrules/resources/Messages.properties

```diff
@@ -362,18 +362,18 @@ pscanrules.usernameidor.soln = Use per user or session indirect object reference
 
 pscanrules.viewstate.content.email.desc = The following emails were found being serialized in the viewstate field:
 pscanrules.viewstate.content.email.name = Emails Found in the Viewstate
-pscanrules.viewstate.content.email.pattern.source = Email pattern - http://www.regular-expressions.info/regexbuddy/email.html
+pscanrules.viewstate.content.email.pattern.source = Email pattern - https://www.regular-expressions.info/regexbuddy/email.html
 pscanrules.viewstate.content.ip.desc = The following potential IP addresses were found being serialized in the viewstate field:
 pscanrules.viewstate.content.ip.name = Potential IP Addresses Found in the Viewstate
 pscanrules.viewstate.content.ip.pattern.source = IP pattern - http://www.regular-expressions.info/examples.html
```

Comment:
This was not updated.

Reply:
https://www.regular-expressions.info/ip.html

Reply:
done